packages/mwp-language-plugin/src/util/getLanguage.js

```diff
@@ -29,7 +29,16 @@ export const getCookieLang: ParseRequestLang = (request: HapiRequest) => {
 export const getUrlLang: ParseRequestLang = (request: HapiRequest) => {
 	const { supportedLangs } = getServerSettings(request);
 	const urlLang = request.url.path.split('/')[1];
-	return supportedLangs.includes(urlLang) && urlLang;
+	const validRequestLang = supportedLangs.includes(urlLang) && urlLang;
+
+	// If request url language is invalid, look at referer for language code
+	// This is for cases when the request url is the api proxy path ie `/mu_api/`
+	if (!validRequestLang && request.headers['referer']) {
```

Comment:
I think the more conventional place to check the referrer header is `request.info.referrer` https://hapijs.com/api#-requestinfo